src/index.js

```diff
@@ -0,0 +1,189 @@
+      '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temp > 69 && state.temp <= 79) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temp > 79) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+const inputCity = (event) => {
+  const city = document.querySelector('#cityName');
+  const displayName = document.querySelector('#cityNameDisplay');
+  displayName.textContent = city.value;
+};
+
+const changeSky = (event) => {
+  const skySelector = document.querySelector('#skySelector');
+  const sky = document.querySelector('#skyDisplay');
+  if (skySelector.value === 'cloudy') {
+    document.querySelector('#skyDisplay').textContent =
+      '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skySelector.value === 'raining') {
+    document.querySelector('#skyDisplay').textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skySelector.value === 'snowing') {
+    document.querySelector('#skyDisplay').textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  } else if (skySelector.value === 'sunny') {
+    document.querySelector('#skyDisplay').textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  }
+};
+
+const loadCityTemp = (event) => {
+  const city = document.querySelector('#cityName');
+  const searchString = city.value;
+  const tempPromise = getTempFromSearch(searchString);
+  const tempCount = document.querySelector('#temperatureDisplay');
+  tempPromise.then((temp) => {
+    state.temp = temp;
+    tempCount.textContent = ` ${state.temp} ℉`;
+    changeTempColor();
+  });
+};
+
+const registerEventHandlers = (event) => {
+  const increaseTemp = document.querySelector('#increaseTemperature');
+  increaseTemp.addEventListener('click', increaseTemperature);
+
+  const decreaseTemp = document.querySelector('#reduceTemperature');
+  decreaseTemp.addEventListener('click', decreaseTemperature);
+
+  const changeColorOnHotter = document.querySelector('#increaseTemperature');
+  changeColorOnHotter.addEventListener('click', changeTempColor);
+
+  const changeColorOnColder = document.querySelector('#reduceTemperature');
+  changeColorOnColder.addEventListener('click', changeTempColor);
+
+  const defaultCity = document.querySelector('#cityReset');
+  defaultCity.addEventListener('click', resetCity);
+
+
+  const newLandscapeOnDecreacse = document.querySelector('#reduceTemperature');
+  newLandscapeOnDecreacse.addEventListener('click', changeLandscape);
+
+  const newCity = document.querySelector('#cityName');
+  newCity.addEventListener('input', inputCity);
+
+
+  const newSky = document.querySelector("#skySelector")
+  newSky.addEventListener("change", changeSky)
+
+  const loadTemperature = document.querySelector('#loadTemperature');
+  loadTemperature.addEventListener('click', loadCityTemp);
+};
+
+document.addEventListener("DOMContentLoaded", registerEventHandlers);
+
+
+
+/* Load sky and landscape after loading. */
+const postLoadFormatting = (event) => {
+  changeSky(event);
```

Comment:
Avoid arbitrarily passing one event type in where a different event is expected. The `postLoadFormatting` function is registered on `DOMContentLoaded`, so the event will have details about the loading being done. But `changeSky` is registered for a select change event, which will have different information. It turns out `changeSky` doesn't actually use the event, so no problem occurs, but keep an eye out for that. Rather than calling event handlers directly, prefer placing the actual logic in a separate function that can be called with explicit data (or no data if no dependency).